db/seed.js

```diff
@@ -6,19 +6,19 @@ let productNames = ['Personalized Cheese Board Wedding Gift Set', 'Unpainted Dor
 let productOptions = [
   {
     optionName: 'Size',
-    choices: ['extra-small', 'small', 'medium', 'large', 'extra-large']
+    choices: [{choice: 'extra-small'}, {choice: 'small'}, {choice: 'medium'}, {choice: 'large'}, {choice:'extra-large'}]
   },
   {
     optionName: 'Material',
-    choices: ['ash', 'walnut', 'ebony', 'aluminum', 'brushed steel', 'glass', 'solid titanium', 'pure gold', 'solid diamond']
+    choices: [{choice: 'ash'}, {choice:'walnut'}, {choice:'ebony'}, {choice: 'aluminum'}, {choice: 'brushed steel'}, {choice: 'glass'}, {choice: 'solid titanium'}, {choice: 'pure gold'}, {choice: 'solid diamond'}]

```

Comment:
Why are we adding a choice key to each element? That'll probably turn into some awkward syntax (`productOptions.choices[0].choice`)